### PR TITLE
Tests for feed endpoint exclude parameter was added

### PR DESCRIPTION
**Pull request by mpushnikA1QA**

### CODE REVIEW

File: src/util/commonUtils.js [hunk 1]
```diff
@@ -23,10 +23,21 @@ function deleteWildcardSearchingSymbol(str) {
 
 const GET_ENTITY_ID = (entity) => `${entity}Id`;
 
+function checkThatTheArrayIsContainedInTheOriginalArray(where, what) {
```
Comment (agromyko) [inline src/util/commonUtils.js:26]: method and parameter names are not clear, with 'check' in the beginning of the method we expect that we'll perform some check and receive boolean for example, however here we just get an array in return, also it's not clear what is 'where' and 'what' - need to be specified if these are some arrays/ object keys

File: src/test/steps/checkExcludeParameter.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    if (diff.length > 0) {
+      arrayForNonMatchingFieldsInTheResponse.push(`There shouldn't be fields [${diff}] for ${this.entityId} - [${item[this.entityId]}]`);
+    }
+  }
+  expectChai(arrayForNonMatchingFieldsInTheResponse.length).to.equal(0, `The fields specified in the request query shouldn't be in the response body : ${arrayForNonMatchingFieldsInTheResponse.toString()}`);
+});
+
+//TODO - It is necessary to do code refactoring for the entire project.
+Then(/^Check that the fields specified in the request query are present in the response body$/, function () {
+  const arrayForNonMatchingFieldsInTheResponse = [];
```
Comment (agromyko) [inline src/test/steps/checkExcludeParameter.js:44]: general comment - if possible please try to shorten the name of variables as here could it be nonMachingFieldsArray for instance? as we already know from the step definition that and 'for' cycle that we are working with response, the entire project will be easier to read then

Comment (mpushnikA1QA) [reply src/test/steps/checkExcludeParameter.js]: Updated

File: src/util/commonUtils.js (hunk 1 above)
Comment (mpushnikA1QA) [reply src/util/commonUtils.js]: Fixed

**Review by agromyko — APPROVED** [review]

**Pull request merged**